### PR TITLE
Add twitter, facebook and clipboard sharing

### PR DESCRIPTION
**Pull request by samuveth**

### CODE REVIEW

File: src/composables/useShare.ts [hunk 1]
```diff
@@ -0,0 +1,60 @@
+import { useI18n } from 'vue-i18n';
+import { useClipboard } from '@vueuse/core';
```
Comment (bonustrack) [inline src/composables/useShare.ts:2]: Would be great to use that in the others place where we use clipboard. And also add a notification to confirm copy worked.

**Comment by bonustrack** [comment]
Would be great to detect the domain so if someone is on vote.balancer.fi it share the same URL.

**Comment by samuveth** [comment]
Do you have twitter on mobile for testing this?

**Comment by bonustrack** [comment]
Here we should add the proposal title when using the share API

**Comment by samuveth** [comment]
> Here we should add the proposal title when using the share API

Is that better now?

**Review by bonustrack — APPROVED** [review]
Works well now

**Pull request merged**